### ISSUE TITLE
Two ThermIQ - one Home Assistant

### ISSUE THREAD

**Issue by bjugan** [ThermIQ/thermiq_mqtt-ha#17]
Hi,

I have two heatpumps and two ThermiIQ. I have given them different node names and both of them appear nicely in MQTT explorer on different nodes. I want to integrate both with the same Home Assistant installation. The first one was easy to set up in HA following the guide. But I'm unsure how to set up the second one without coming in conflict with the first on. I would be greatful for any tips :-) (PS! I'm a HA beginner).

Marius

**Comment by ThermIQ** [comment]
This is currently not supported as I haven't figured out how to do it yet. It comes down to having a platform and integration in the same directory. I'll put it on my to do list as I now know there's an interrest for it :-)

**Comment by ThermIQ** [comment]
The internal storage of states is currently hardcoded as well but thats an easier fix, then maybe it could work duplicating thermiq:mqtt in the custom_components folder.

**Comment by bjugan** [comment]
Thanks, I guess this is a very unusual need so I don't expect anything :-) I can solve this with a second HA installation.

**Comment by ThermIQ** [comment]
On step forward. All variables are now based on DOMAIN which is set in __init_py__

Duplicating the custom_components/thermiq_mqtt directory and changing DOMAIN could work...

**Comment by bjugan** [comment]
Sorry for my very late follow up. And thanks for fixing this!

It's almost working perfect. All read only values are show correct including the pump graphics,
But if I add both pumps one of the dashboard cards only shows -1 in all input_numbers.

![image](https://user-images.githubusercontent.com/35533010/151054454-7eaecc46-c03a-4e9b-a6a4-ecdaf4ba6c14.png)

If I setup one integration (doesn't matter which one) in configuration.yaml everything works perfect, but as soon as I add the second one in configuration.yaml this happens on one of the pumps (same one each time).

What I have done:
- Duplicated thermiq_mqtt folder and renamed folders to thermiq_mqtt_hytta and thermiq_mqtt_hjemme
- init.py, line 9
   custom_components.thermiq_mqtt_hytta.thermiq_regs
   custom_components.thermiq_mqtt_hjemme.thermiq_regs
- init.py, line 94 
   set domain to thermiq_mqtt_hytta and thermiq_mqtt_hjemme
- manifest .json
   set domain to thermiq_mqtt_hytta and thermiq_mqtt_hjemme
- configuration.yaml
```
thermiq_mqtt_hytta:
  mqtt_node: ThermIQ/ThermIQ-mqtt-B150
  language: "en"
  thermiq_dbg: False

thermiq_mqtt_hjemme:
  mqtt_node: ThermIQ/ThermIQ-mqtt-bb
  language: "en"
  thermiq_dbg: False
```
- Updated lovelace configs with domains

I find this error in the log even if the entities exists (more then 100 000 occurrences since yesterday before I emptied the log):
![image](https://user-images.githubusercontent.com/35533010/151059260-b36f7ca8-d948-43bf-9332-aa931d322a06.png)

I have tried to look into the code but I'm not familier with either python or Home Assistant development.

I'm running Home Assistant OS 7.2, core-2021.12.10, supervisor-2021.12.2, HACS 1.19.3, ThermIQ MQTT 1.3.2

Any tips? :-)

**Comment by ThermIQ** [comment]
Can you see that both input_number.thermiq_mqtt_hjemme_brine_run_in_t and corresponding hytte exists?

does it change if you change order of them in configuration.yaml

are there any input_number.thermiq_mqtt_xxx only

There is also the services.yaml file. You might have to add hytte/hjemme to the service names as well.

**Comment by bjugan** [comment]
> Can you see that both input_number.thermiq_mqtt_hjemme_brine_run_in_t and corresponding hytte exists?

![image](https://user-images.githubusercontent.com/35533010/151063593-6e5379ba-7d75-46a2-9d7f-d05dfd09f129.png)
 
> does it change if you change order of them in configuration.yaml

I have tried. Didn't help. And it's still the "hjemme" pump that gets -1 values. 
 
> are there any input_number.thermiq_mqtt_xxx only

You mean like this? Guess it's "left overs" from the time I only integrated one pump.

![image](https://user-images.githubusercontent.com/35533010/151064338-6905f5d4-4120-45c8-98d2-f6c49394474b.png)


> There is also the services.yaml file. You might have to add hytte/hjemme to the service names as well.

Thanks, I'll look into this and report back. Need to read more about  HA development to understand what I should look for so it may take time :-)
